source/UberRides/Resources/pt-br.lproj/Localizable.string

```diff
@@ -0,0 +1,27 @@
+/* The title of the request button that deep links to the Uber app */
+"Ride there with Uber" = "Chegue lá de Uber";
+"Get a ride" = "Peça um Uber";
+"Sign in with Uber" = "Login com Uber";
+"There was a problem authenticating you." = "Tivemos um problema autenticando você.";
+"The Ride Request Widget encountered a problem." = "Widget do Uber encontrou um problema.";
+"OK" = "OK";
+"Done" = "Feito";
+"Invalid Client ID provided." = "Client ID inválido";
+"Invalid Redirect URI provided." = "URL de redirecionamento inválida";
+"The server was unable to understand your request." = "Nosso servidor não entendeu seu pedido.";
+"Unable to interpret the response from the server." = "Impossível de interpretar a resposta do servidor.";
+"Your app is not authorized for the requested scopes." = "Seu aplicativo não está autorizado a fazer chamadas neste escopo.";
+"The Redirect URI provided did not match what was expected." = "A URI fornecida não corresponde ao esperado.";
+"A server error occurred." = "Erro inesperado de servidor.";
+"Unable to present the login view." = "Impossível de apresentar a tela de login.";
+"Unable to save the access token." = "Impossível de salvar token de acesso.";
+"Login is temporarily unavailable." = "Login temporariamente indisponível.";
+"User cancelled the login process." = "Usuário cancelou processo de login.";
+"Try Again" = "Tente novamente";
+"Cancel" = "Cancelar";
+"The operation you are attempting is not supported on the current device." = "A operação não pode ser executado neste tipo de equipamento.";
+"%d min away" = "%d min longe";
+"%d mins away" = "%d mins longe";
```

Comment:
yes you can. We had bad time trying to fit a better world here. The problem is the length. 
But I think just "mins" is comprehensible.